assets/i18n/qqq.json

```diff
@@ -355,6 +355,7 @@
 	"I18N_PLAYER_LOADING": "Text displayed in the middle of the page while the content is being loaded.\n{{Identical|Loading}}",
 	"I18N_PLAYER_NEXT_LESSON": "Text displayed when the user finishes an exploration played within a collection. - Text shown above the next exploration the user should play in the collection.",
 	"I18N_PLAYER_NO_OBJECTIVE": "Text displayed when the user is playing an exploration. - When the user clicks the information button, a dialog opens with further information. This text is shown in that dialog when the author has not entered a description for the exploration.",
+	"I18N_PLAYER_IS_PRIVATE": "Text displayed when the user is playing an exploration. - When the user clicks the information button, a dialog opens with further information. This text is shown in that dialog when the author has not published the exploration and is private.",
```

Comment:
... the exploration, i.e. the exploration is private.

(Your current formulation suggests that the author is private.)

Reply:
okay, done